app/mailers/user_mailer.rb

```diff
@@ -92,7 +92,7 @@ def subject_for(key)
       default: [:subject, key.to_s.humanize], app_name: app_name)
   end
 
-  def preview?
-    GovukAdminTemplate.environment_label == "Preview"
+  def integration?
+    Plek.environment == "integration"
```

Comment:
These string comparisons will proliferate forever. Maybe we could clean these up a bit by having Plek answer to `Plek.environment.{integration?,production?,development?,other?}`